### PR TITLE
feat : added alerts feature

### PR DESCRIPTION
**Pull request by vivekgudipudi**
added alerts feature and refactored introduction page

### CODE REVIEW

File: CSS/introduction.css [hunk 1]
```diff
@@ -17,26 +17,51 @@
 }
 
 .main-content {
-    max-width: 75%;
+    /* max-width: 75%; */
```
Comment (nishasen) [inline CSS/introduction.css]: remove the comments

File: CSS/introduction.css [hunk 2]
```diff
@@ -17,26 +17,51 @@
...
+    width: 70%;
+    height: max-content;
     margin: var(--xs);
-    border: 2px solid black;
```
Comment (nishasen) [inline CSS/introduction.css:22]: use variables instead of giving direct colors

File: CSS/introduction.css [hunk 3]
```diff
@@ -17,26 +17,51 @@
...
+    justify-content: center;
 }
 
 .main-content-landing-page {
     max-width: 48%;
     margin: var(--xs);
 }
 
+.main-content-example-box{
+    background-color: #f3f4f6;
```
Comment (nishasen) [inline CSS/introduction.css]: instead of using color codes or names directly, use variables

File: CSS/utilities.css [hunk 4]
```diff
@@ -37,6 +41,9 @@ body{
     /* --text-color:#e5e7eb ; */
 
     --primary-color: #6B63FF;
+    --danger-color : #F44336;
+    --success-color :#4CAF50;
+    --warning-color : #FF9800;
```
Comment (nishasen) [inline CSS/utilities.css:46]: define colors in different variables folder, and call the variables in utilities

**Review by nishasen — APPROVED** [review]
I have added some comments please look into it, and add a screenshot or video of the designs you have created in the comments for better clarity.

File: CSS/introduction.css (hunk 2 above)
Comment (vivekgudipudi) [reply CSS/introduction.css:22]: that border is for testing responsiveness.. i'll remove it after making components

File: CSS/introduction.css (hunk 3 above)
Comment (vivekgudipudi) [reply CSS/introduction.css]: okay, sure

File: CSS/utilities.css (hunk 4 above)
Comment (vivekgudipudi) [reply CSS/utilities.css:46]: okay

File: CSS/introduction.css (hunk 3 above)
Comment (pariyar07) [inline CSS/introduction.css]: try using color variables

File: CSS/introduction.css [hunk 5]
```diff
@@ -17,26 +17,51 @@
...
+    border-radius:var(--xs);
+    /* border: 1px solid; */
+    margin: var(--md);
+    padding: var(--sm);
+    display: flex;
+    align-items: center;
```
Comment (pariyar07) [inline CSS/introduction.css:40]: make a flex utility class and use it

File: components/alerts/alerts.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+<body>
+    <header class="nav">
+        <nav class="nav-header">
+            <div class="t2">
+                <a href="/index.html">Keviv UI</a>
+            </div>
+            <div class="nav-components">
+                <ul class="nav-component-list">
+                    <li class="nav-pills"><a href="#" class="active">Alert</a></li>
+                    <li class="nav-pills"><a href="#">Avatar</a></li>
```
Comment (pariyar07) [inline components/alerts/alerts.html:25]: add links to other pages

**Comment by pariyar07** [comment]
Try to add a screenshot of the UI to make it easy for the review.

File: CSS/introduction.css (hunk 3 above)
Comment (vivekgudipudi) [reply CSS/introduction.css]: changed

File: CSS/introduction.css (hunk 5 above)
Comment (vivekgudipudi) [reply CSS/introduction.css:40]: thought of using the same class in all the pages, so I placed the pre defined class in introduction.css page

File: components/alerts/alerts.html (hunk 6 above)
Comment (vivekgudipudi) [reply components/alerts/alerts.html:25]: will add once I start working on other pages, I haven't created those pages yet

**Review by pariyar07 — APPROVED** [review]

**Pull request merged**